examples/pytorch/CaptumExample/Titanic_Captum_Interpret.py

```diff
@@ -0,0 +1,394 @@
+        net,
+        train_features,
+        train_labels,
+        test_features,
+        test_labels,
+        feature_names,
+    )
+
+
+def train_step(train_features):
+    train_input_tensor = torch.from_numpy(train_features).type(torch.FloatTensor)
+    out_probs = net(train_input_tensor).detach().numpy()
+    out_classes = np.argmax(out_probs, axis=1)
+    try_mlflow_log(
+        mlflow.log_metric,
+        "Train Accuracy",
+        float(sum(out_classes == train_labels) / len(train_labels)),
+    )
+    print("Train Accuracy:", sum(out_classes == train_labels) / len(train_labels))
+    return train_input_tensor
+
+
+def test_step(test_features):
+    test_input_tensor = torch.from_numpy(test_features).type(torch.FloatTensor)
+    out_probs = net(test_input_tensor).detach().numpy()
+    out_classes = np.argmax(out_probs, axis=1)
+    try_mlflow_log(
+        mlflow.log_metric,
+        "Test Accuracy",
+        float(sum(out_classes == test_labels) / len(test_labels)),
+    )
+    print("Test Accuracy:", sum(out_classes == test_labels) / len(test_labels))
+    return test_input_tensor
+
+
+def feature_conductance(test_input_tensor):
+    """
+    Beyond just considering the accuracy of the classifier, there are many important questions to understand how the model is working and it's decision, which is the purpose of Captum, to help make neural networks in PyTorch more interpretable.
+    The first question we can ask is which of the features were actually important to the model to reach this decision?
+    This is the first main component of Captum, the ability to obtain **Feature Attributions**.
+    For this example, we will apply Integrated Gradients, which is one of the Feature Attribution methods included in Captum.
+    More information regarding Integrated Gradients can be found in the original paper here: https://arxiv.org/pdf/1703.01365.pdf
+
+    To apply integrated gradients, we first create an IntegratedGradients object, providing the model object.
+
+    To compute the integrated gradients, we use the attribute method of the IntegratedGradients object. The method takes tensor(s) of input examples (matching the forward function of the model), and returns the input attributions for the given examples. For a network with multiple outputs, a target index must also be provided, defining the index of the output for which gradients are computed. For this example, we provide target = 1, corresponding to survival.
+    The input tensor provided should require grad, so we call requires_grad on the tensor. The attribute method also takes a baseline, which is the starting point from which gradients are integrated. The default value is just the 0 tensor, which is a reasonable baseline / default for this task.
+    The returned values of the attribute method are the attributions, which match the size of the given inputs, and delta, which approximates the error between the approximated integral and true integral.
+
+    """
```

Comment:
Do we really need this explanation? Can we just mention in README that this example is based on https://captum.ai/tutorials/Titanic_Basic_Interpret?

Reply:
@harupy , Yeah we can do that. I have the code now.